### PR TITLE
Update supported versions

### PR DESCRIPTION
**Pull request by zerolab**
Django <= 3.1
django-graphene < 2.14.0

Need to fix

```
graphene_django/types.py:110: UserWarning: Excluding the custom field "url" on DjangoObjectType "HomePage" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "url" on DjangoObjectType "AuthorPage" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "url" on DjangoObjectType "BlogPage" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "id" on DjangoObjectType "Person" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "id" on DjangoObjectType "Advert" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "id" on DjangoObjectType "BlogPageRelatedLink" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "id" on DjangoObjectType "Author" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "id" on DjangoObjectType "SocialMediaSettings" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "id" on DjangoObjectType "CustomImage" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "width" on DjangoObjectType "CustomImage" has no effect. Either remove the custom field or remove the field from the "exclude" list.
graphene_django/types.py:110: UserWarning: Excluding the custom field "height" on DjangoObjectType "CustomImage" has no effect. Either remove the custom field or remove the field from the "exclude" list.
```

which is a result of https://github.com/graphql-python/graphene-django/pull/873

tl;dr these are of the result of https://github.com/GrappleGQL/wagtail-grapple/blob/master/grapple/actions.py#L246 -- `get_fields_and_properties()` will return all fields and properties, which include "id", "pk" for Page subclasses (defined as custom in PageInterface), "url", "width" and "height" for Images (defined in BaseImageObjectType)

### CODE REVIEW

**Comment by zerolab** [comment]
Hey @ruisaraiva19, would you be able to cast an eye on this? I tested locally with Django 3.1.1, graphene-django 2.13.0 and graphql-core 2.3.2

**Comment by zerolab** [comment]
Will need to work out the exclusion logic as it throws warnings for things like snippets too (e.g. `Django model "missions.PointOfStudy" does not have a field or attribute named "pointofstudyorderable". Consider removing the field from the "exclude" list of DjangoObjectType "PointOfStudy" because it has no effect`) when you do not directly add them to `grapple_fields`

**Comment by zerolab** [comment]
@ruisaraiva19 @NathHorrigan I'd like to get this in so we make a new release soon.
I've opened #123 as a followup

**Review by ruisaraiva19 — APPROVED** [review]
